src/api/searchApi.ts

```diff
@@ -8,11 +8,15 @@ export enum SearchApiPaths {
 
 export const searchRegistrations = async (
   searchTerm?: string,
-  numberOfResults?: number,
-  searchAfter?: string,
+  numberOfResults = 10,
+  searchAfter = 0,
   cancelToken?: CancelToken
 ) => {
-  const url = searchTerm ? `${SearchApiPaths.REGISTRATIONS}?query=${searchTerm}` : SearchApiPaths.REGISTRATIONS;
+  const url = searchTerm
+    ? `${SearchApiPaths.REGISTRATIONS}?query=${searchTerm}&results=${numberOfResults}&from=${searchAfter}`
+    : numberOfResults !== 10 || searchAfter !== 0
```

Comment:
hva oppnås egentlig med denne?
hvis `numberOfResults = 10`  og `searchAfter = 0`, så droppes paging?
burde det ikke heller være `numberOfResults = -1` elns om man evt vil ha alt (noe jeg ikke tror man noen gang vil ha)

Reply:
Det er defaultverdier. Men de settes kanskje i backend uansett når man gjør fritekst-søk? Paging droppes ikke når de er satt nei. Pagineringskomponenten skal vises hele tiden. Men det er ulike endepunkt for «alt» og når man søker med en søketerm.

Reply:
tror ikke de settes i backend uansett nei, så hvis dette blir `false` går det til linje 19 som ikke bruker paginering(?)

Reply:
altså paginerings-komponenten vises fortsatt ja, men vi henter alt uten paginering fra backend.
Det jeg ikke forstår er hvorfor vi ikke vil bruke paginering mot backend når akkurat `numberOfResults !== 10 || searchAfter !== 0` er `false`

Reply:
Ja, skjønner at det er forvirrende. Burde kanskje gjort det annerledes. Det har ikke noe med paginering å gjøre, bare at numberOfResults og searchAfter er annerledes enn defaultverdiene, så vi må kjøre requesten mot backend på nytt (må oppdatere request-urlen). Men kanskje unødvendig med defaultverdier. Skal sjekke

Reply:
Mer forståelig nå? https://github.com/BIBSYSDEV/NVA-Frontend/pull/1367/commits/7e2c645ac967626a569620686b3f4f4780c596ce

Reply:
jess, nå henger jeg med!